src/test/java/Homework17.java

```diff
@@ -0,0 +1,58 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        String expectedSongAddedMessege = "Added 1 song into \"Test Pro Playlist.\" ";
+        navigateToPage();
+        provideEmail("zhanna.ivanova@testpro.io");
+        providePassword("12345678");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchSong("rainday");
+        clickViewAllBtn();
+        selectFirstSongResult();
+        clickAddToBtn();
+        choosePlaylist();
+        Thread.sleep(2000);
+        Assert.assertEquals(getAddToPlayListSuccessMsg(), expectedSongAddedMessege);
+    }
+
+    public void searchSong(String name) throws InterruptedException {
+        WebElement searchField = driver.findElement(By.cssSelector("div#searchForm input[type='search']"));
+        searchField.sendKeys(name);
+        Thread.sleep(2000);
+    }
+    public void clickViewAllBtn() throws InterruptedException {
+        WebElement viewAll = driver.findElement(By.xpath("//button[@data-test='view-all-songs-btn']"));
+        viewAll.click();
+        Thread.sleep(2000);
+    }
+    public void selectFirstSongResult() throws InterruptedException {
+        WebElement firstSong = driver.findElement(By.xpath("//section[@id='songResultsWrapper']//tr[@class='song-item'][1]"));
+        firstSong.click();
+        Thread.sleep(2000);
+    }
+
+    public void clickAddToBtn () throws InterruptedException {
+        Thread.sleep(2000);
+            WebElement addToButton = driver.findElement(By.xpath("//section[@id='songResultsWrapper']//button[@data-test='add-to-btn']"));
+            addToButton.click();
+            Thread.sleep(2000);
+        }
+        public void choosePlaylist() throws InterruptedException {
+            WebElement playlist  = driver.findElement(By.xpath("//section[@id='songResultsWrapper']//li[contains(text(),'Love')]"));
```

Comment:
Hello,
Your homework 17 has been reviewed, but the script Homework17 execute failed
![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/144403954/3dee5c2b-d35b-4bd4-8da9-b955946f213a)

At step `choosePlaylist` , you click playlist has name contanis Love but when at step verify expectedSongAddedMessege you leave the playlist name `Test Pro Playlist`. So when comparing it will return false

//HINT: You need to agree on the playlist name in step choosePlaylist và expectedSongAddedMessege